apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -163,6 +172,18 @@ export default class WorkshopForm extends React.Component {
     });
   }
 
+  loadRegionalPartners() {
+    this.loadRegionalPartnersRequest = $.ajax({
+      method: "GET",
+      url: '/api/v1/regional-partners/for_user',
```

Comment:
As discussed offline, please change this route in a followup PR to be `/api/v1/regional-partners`, switch the action from `for_user` to `index`, and rename the existing `index` action something like `for_district_and_course` since it's less standard / more specific.